### ISSUE TITLE
Inquiry regarding Dataset Splits and Reproduction of Baseline Results in Table 2

### ISSUE THREAD

**Issue by Xyu-o-o** [worldbench/Pi3DET#3]
Thank you for your impressive work and for releasing the Pi3DET dataset. We are very interested in conducting further research based on your dataset and framework.
I am currently trying to reproduce the results for the cross-platform adaptation tasks (e.g., Vehicle → Drone/Quadruped) as shown in Table 2. However, after downloading the released data, I could not find the specific file lists or scripts for the Training/Validation/Test splits for each platform (Vehicle, Drone, and Quadruped).
To ensure my reproduction is consistent with your work, could you please clarify the following points:
1.Dataset Splits: Could you provide the official .pkl or .txt split files for the Vehicle, Drone, and Quadruped platforms? Specifically, I would like to know the exact frame IDs used for training and validation in each subset.
2.Experimental Pipeline: For the results in Table 2, is my understanding of the training/evaluation process correct?
        PA Stage: Trained on the Source (Vehicle) training set with Random Platform Jitter.
        KA Stage: Jointly trained on the Source (Vehicle) training set (with GT) and the Target (e.g., Drone) training set (with pseudo-labels).
        Evaluation: The final metrics (AP) are reported on the Target (e.g., Drone) validation set.
3.Baseline Reproduction (Source / Target / Combined All): To accurately reproduce the three baseline rows in Table 2, I believe it is essential to follow the same data partition you used. Could you confirm how these rows were obtained? Specifically:
        Source Platform: Is this the "Source-Only" baseline (trained on the Source training set and directly evaluated on the Target validation set)?
        Target Platform: Is this the "Oracle" result (trained on the Target training set with full GT and evaluated on the same Target validation set)?
        Combined All: Does this represent the performance when training on the combined training sets of all platforms with full GT?
Without the official split information, it is difficult to ensure that my training and evaluation are conducted on the same data distributions as yours, which is critical for a fair comparison.
We look forward to your insights and appreciate your guidance on these details. Thank you for your time and for sharing this significant work!

**Comment by AlanLiangC** [comment]
Thanks for your interests.
For Q1&3:, your understanding of the training/evaluation process is correct.
- nuscenes --> drone
```
INFO_SEQUENCES: {
    'train': [
      processed/Falcon/Outdoor_Day/penno_parking_1,
    ],
    'test': [
      processed/Falcon/Outdoor_Day/penno_parking_2,
      ],
}
```
- nuscenes --> quadruped
```
INFO_SEQUENCES: {
    'train': [
      processed/Spot/Outdoor_Day/penno_short_loop,
    ],
    'test': [
      processed/Spot/Outdoor_Night/penno_short_loop,
      processed/Spot/Outdoor_Day/srt_green_loop
      ],
}
```
- Pi3DET --> quadruped/drone
```
# For Car
INFO_SEQUENCES: {
    'train': [
        processed/Car/Urban_Day/car_urban_day_city_hall,
        processed/Car/Urban_Day/penno_big_loop,
        processed/Car/Urban_Day/rittenhouse,
        processed/Car/Urban_Day/ucity_small_loop,
        ],
    'test': [processed/Car/Urban_Day/car_urban_day_city_hall],
}
```

For Q2, yes, you can follow st3d/++'s source training process with our Random Platform Jitter.

**Comment by Xyu-o-o** [comment]
> Thanks for your interests. For Q1&3:, your understanding of the training/evaluation process is correct.
> 
> * nuscenes --> drone
> 
> ```
> INFO_SEQUENCES: {
>     'train': [
>       processed/Falcon/Outdoor_Day/penno_parking_1,
>     ],
>     'test': [
>       processed/Falcon/Outdoor_Day/penno_parking_2,
>       ],
> }
> ```
> 
> * nuscenes --> quadruped
> 
> ```
> INFO_SEQUENCES: {
>     'train': [
>       processed/Spot/Outdoor_Day/penno_short_loop,
>     ],
>     'test': [
>       processed/Spot/Outdoor_Night/penno_short_loop,
>       processed/Spot/Outdoor_Day/srt_green_loop
>       ],
> }
> ```
> 
> * Pi3DET --> quadruped/drone
> 
> ```
> # For Car
> INFO_SEQUENCES: {
>     'train': [
>         processed/Car/Urban_Day/car_urban_day_city_hall,
>         processed/Car/Urban_Day/penno_big_loop,
>         processed/Car/Urban_Day/rittenhouse,
>         processed/Car/Urban_Day/ucity_small_loop,
>         ],
>     'test': [processed/Car/Urban_Day/car_urban_day_city_hall],
> }
> ```
> 
> For Q2, yes, you can follow st3d/++'s source training process with our Random Platform Jitter.

Hi, thank you so much for your quick and helpful reply!

To ensure my setup perfectly matches your work in Table 2 and Table 12, may I clarify a couple of small details regarding the data?

Regarding the Drone and Quadruped platforms, may I confirm if all experiments in the paper involving Pi3DET data only used the specific sequences you listed above? I noticed there are some other sequences in the dataset that weren't mentioned, so I assume they were not involved in the experiments.

Also, for the "Target Platform" baseline in Table 12, I'm curious if the ''car_urban_day_city_hall'' sequence is included in the evaluation, since it appears in both the train and test lists.

If you happen to have the official .txt or .yaml split files available, would it be possible to share them? It would be a great help to avoid any manual errors. Thanks again for your patience and for this great work!
